src/Utilities/Compiler/Extensions/StringCompatExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if !NETCOREAPP
+
+namespace System
+{
+    internal static class StringCompatExtensions
+    {
+        public static bool Contains(this string str, string value, StringComparison comparisonType)
+        {
+            return str.IndexOf(value, comparisonType) >= 0;
```

Comment:
I thought that wasn't good to use the `IndexOf` when we can do `Contains` (see https://github.com/dotnet/roslyn-analyzers/pull/3594).

Reply:
There is no `Contains` method that takes a `StringComparison`.

Reply:
Oh yes of course! Sorry!